src/libraries/System.Private.Xml/src/System/Xml/Serialization/Types.cs

```diff
@@ -1033,13 +1033,16 @@ internal static MemberMapping[] GetAllMembers(StructMapping mapping)
 
         internal static void GetAllMembers(StructMapping mapping, List<MemberMapping> list)
         {
+            // Order matters. Legacy behavior gives defference to base mappings. So we start there.
+            // But we want to replace base mappings with overrides from the derived class.
```

Comment:
Typo: defference -> deference

Reply:
Lol. Co-pilot already showed off it's superior spelling skillz.